deploy-service/common/src/main/java/com/pinterest/deployservice/handler/ConfigHistoryHandler.java

```diff
@@ -44,8 +44,9 @@
 
 public class ConfigHistoryHandler {
     private static final Logger LOG = LoggerFactory.getLogger(ConfigHistoryHandler.class);
-    private static final String CHANGEFEED_TEMPLATE = "{\"type\":\"%s\",\"environment\":\"%s\",\"description\":\"%s\",\"author\":\"%s\","
-            + "\"automation\":\"%s\",\"source\":\"Teletraan\",\"optional-1\":\"%s\",\"optional-2\":\"%s\"}";
+    private static final String CHANGEFEED_TEMPLATE =
+        "{\"type\":\"%s\",\"environment\":\"%s\",\"url\":\"%s\",\"description\":\"%s %s\",\"author\":\"%s\","
+            + "\"automation\":\"%s\",\"source\":\"Teletraan\",\"nimbus_uuid\":\"%s\"}";
```

Comment:
It'd be nice if this can be formatted in the same way it is in the `DeployHandler`.